taprpc/mintrpc/mint.proto

```diff
@@ -62,6 +62,11 @@ message MintAsset {
     This asset will be minted with the same group key as the anchor asset.
     */
     string group_anchor = 6;
+
+    /*
+    The version of asset to mint.
+    */
+    taprpc.AssetVersion asset_version = 7;
```

Comment:
Do we need this field here? I don't see the value in being able to mint an old asset version. Do we mean to signify a different asset type instead perhaps?

Reply:
So we want to sort of leave the path open for either making this version default, or continuing to use the old version. So far I imagine this'll mainly be used when doing more custom higher level protocols, that'll use the vPSBT APIs directly.